### PR TITLE
PM not required

### PR DESCRIPTION
**Pull request by zachthompson**
We don't require a Perl module any longer, e.g. fatheads/longtails, even dev for spices/goodies it's ok for it to be blank.

Definitely shouldn't be kicking out IAs without them.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ install:
       DDG::SpiceBundle::OpenSourceDuckDuckGo
       DDG::FatheadBundle::OpenSourceDuckDuckGo
       DDG::LongtailBundle::OpenSourceDuckDuckGo
-  - dzil authordeps | grep -ve '^\W' | xargs -n 5 -P 10 cpanm --quiet --notest
+  - dzil authordeps | grep -ve '^\W' | xargs -n 5 -P 10 cpanm --mirror http://cpan.org --mirror http://duckpan.org --quiet --notest
```
Comment (GuiltyDolphin) [inline .travis.yml]: Why's this here?

Comment (zachthompson) [inline .travis.yml]: Because that's where UploadToDuckPAN lives now.

Comment (GuiltyDolphin) [inline .travis.yml]: I mean why's it in this PR? The change seems a bit off-topic for the removal of the PM requirement.

Comment (zachthompson) [inline .travis.yml]: Check the Travis build history.

Comment (GuiltyDolphin) [inline .travis.yml]: Ah, okay - it was just breaking the build regardless? Fair enough.

**Comment by moollaza** [comment]
@zachthompson Thanks! This LGTM :+1:

**Pull request merged**